### ISSUE TITLE
Use REST SDK alongside Adaptive Payments?

### ISSUE THREAD

**Issue by tbhockey** [paypal/PayPal-Ruby-SDK#101]
Is there anyway to use the REST SDK and Adaptive Payments (https://github.com/paypal/adaptivepayments-sdk-ruby) in the same app? We've been using Adaptive Payments but want to switch some payment processes to the REST API, but currently if we simply add the rest sdk gem to the gemfile and try to use adaptive payments we run into are conflicts.

```
uninitialized constant VERSION

/gems/paypal-sdk-rest-1.1.2/lib/paypal-sdk/core/api/base.rb:157:in `user_agent'
/gems/paypal-sdk-rest-1.1.2/lib/paypal-sdk/core/api/base.rb:69:in `default_http_header'
/gems/paypal-sdk-adaptivepayments-1.105.0/lib/paypal-sdk/adaptive_payments/api.rb:16:in `default_http_header'
/gems/paypal-sdk-rest-1.1.2/lib/paypal-sdk/core/api/base.rb:79:in `api_call'
/gems/paypal-sdk-rest-1.1.2/lib/paypal-sdk/core/api/base.rb:96:in `post'
/gems/paypal-sdk-adaptivepayments-1.105.0/lib/paypal-sdk/adaptive_payments/services.rb:155:in `Pay'
```

You can see in the trace above it attempts to use `paypal-sdk-adaptivepayments`, but calling `Pay` within that then calls methods from `paypal-sdk-rest`.

We've namespaced the API's into separate modules in our app, but they still conflict. Is there anyway to instantiate these 2 API's so they work separately from one another? Perhaps 2 different `paypal.yml` files?

**Comment by jziaja** [comment]
Hi @tbhockey , thanks for reporting this!  This sounds like a bug with this SDK as it should be allowed to work alongside the Adaptive Payments SDK since support for that API is not yet available via REST.

Just curious, but can you share what module namespacing changes you attempted with both this SDK and Adaptive Payments that didn't work?  The conflict you mentioned looks to be originating from the recently-merged Core SDK code, and the module namespaces look to have been left unchanged (e.g. **base.rb** looks to be identical between [this SDK](https://github.com/paypal/PayPal-Ruby-SDK/blob/master/lib/paypal-sdk/core/api/base.rb#L1) and [the Core SDK](https://github.com/paypal/sdk-core-ruby/blob/master/lib/paypal-sdk/core/api/base.rb#L1)).

**Comment by tbhockey** [comment]
We've tried various namespaces, what I have at the moment is:

```
module PayPalRest
  require 'paypal-sdk-rest'
  include PayPal::SDK::REST
end

module PayPalAdaptive
  # some custom convenience methods here.
end
```

**Comment by tbhockey** [comment]
Perhaps it's worth mentioning that using this SDK works fine, it's just using the Adaptive Payments SDK is where it chokes.

**Comment by jwilsjustin** [comment]
This issue seems related to #92

**Comment by tbhockey** [comment]
@jwilsjustin @jziaja This fork https://github.com/jwilsjustin/PayPal-Ruby-SDK appears to fix the problem.

**Comment by juwlee** [comment]
@tbhockey this should have been fixed with #102 merge. (Thanks @jwilsjustin !)
